### DISCUSSION TITLE
STREAMLITE

### DISCUSSION THREAD

**Discussion by ikiteloo** [halimkun/blogspot-url-grabber#14] [Q&A] [answered]
Streamlite bisa dipakai untuk python agar bisa membbuatnya layaknya web

**Comment by halimkun** [comment, marked as answer]
bisa, tapi saya tidak tahu apa istilahnya
namun program python anda akan bisa di render dan ditampilkan di halaman web layaknya html, css, dan js

**Reply by ikiteloo** [reply]
Oke, terimakasih untuk gambranya
